app/src/main/kotlin/nl/eduid/screens/personalinfo/PersonalInfoScreen.kt

```diff
@@ -68,164 +74,359 @@ fun PersonalInfoScreen(
-        val context = LocalContext.current
-        AlertDialogWithSingleButton(
-            title = errorData.title(context),
-            explanation = errorData.message(context),
-            buttonLabel = stringResource(R.string.Button_OK_COPY),
-            onDismiss = dismissError
+    Column(
+        modifier = modifier
+            .padding(it)
+            .systemBarsPadding()
+            .padding(horizontal = 24.dp),
+        verticalArrangement = Arrangement.spacedBy(16.dp)
+    ) {
+        Text(
+            style = MaterialTheme.typography.titleLarge.copy(
+                textAlign = TextAlign.Start, color = MaterialTheme.colorScheme.onSecondary
+            ),
+            text = stringResource(R.string.Profile_Title_COPY),
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(top = 16.dp)
+        )
+        Text(
+            style = MaterialTheme.typography.bodyLarge,
+            text = stringResource(R.string.Profile_Info_COPY),
+        )
+        if (!personalInfo.isVerified) {
+            NotVerifiedIdentity(
+                selfAssertedName = personalInfo.seflAssertedName,
+                isLoading = isLoading,
+                addLinkToAccount = addLinkToAccount,
+                onNameClicked = onNameClicked
+            )
+        } else {
+            VerifiedIdentity(
+                personalInfo = personalInfo,
+                isLoading = isLoading,
+                onNameClicked = onNameClicked,
+                openVerifiedInformation = openVerifiedInformation
+            )
+        }
+
+        Text(
+            style = MaterialTheme.typography.titleLarge.copy(color = MaterialTheme.colorScheme.onSecondary),
+            text = stringResource(R.string.Profile_ContactDetails_COPY),
+            modifier = Modifier.padding(top = 16.dp)
+        )
+        InfoField(title = personalInfo.email,
+            subtitle = stringResource(R.string.Profile_Email_COPY),
+            modifier = Modifier.clickable {
+                onEmailClicked()
+            })
+
+        if (personalInfo.institutionAccounts.isNotEmpty()) {
+            RoleAndInstitutions(openVerifiedInformation, personalInfo.institutionAccounts)
+        }
+
+        LinkAccountCard(
+            title = R.string.Profile_AddRoleAndInstitution_COPY,
+            subtitle = R.string.Profile_AddViaSurfconext_COPY,
+            enabled = !isLoading,
+            addLinkToAccount = addLinkToAccount
         )
+        val configuration = LocalConfiguration.current
+        Surface(
+            modifier = Modifier
+                .requiredWidth(configuration.screenWidthDp.dp)
+                .background(MaterialTheme.colorScheme.tertiaryContainer)
+                .padding(vertical = 12.dp, horizontal = 24.dp),
+        ) {
+            OutlinedButton(
+                onClick = { onManageAccountClicked(personalInfo.dateCreated.getDateTimeString("EEEE, dd MMMM yyyy 'at' HH:MM")) },
```

Comment:
I'm not sure what this string is, but hardcoding the english 'at' seems to be incorrect